### ISSUE TITLE
HiddenVariable false positive

### ISSUE THREAD

**Issue by gquerret** [Riverside-Software/sonar-openedge#128]
A variable with the same defined in the getter of a property and in any other method trigger a false positive in the HiddenVariable rule.

```
  define public property prop1 as char no-undo
  get:
    define variable hdl1 as handle no-undo.
    return ''.
  end get.
  
  method public void method1 ():
    define variable hdl1 as handle no-undo.  // Not hiding the previous hdl1 variable
    return.
  end method.

```

**Comment by gquerret** [comment]
Fixed
